routersploit/interpreter.py

```diff
@@ -159,6 +159,7 @@ class RoutersploitInterpreter(BaseInterpreter):
     help                        Print this help menu
     use <module>                Select a module for usage
     exec <shell command> <args> Execute a command in a shell
+    search <search term>        Search for appropriate module
```

Comment:
I added the complete feature! Thank you for your help.

However, after fixing the tests, a number of errors (not fails) popped while testing my commit.  I can't fix it...could you look at the errors?